test/jasmine/tests/scatter_test.js

```diff
@@ -539,6 +539,42 @@ describe('end-to-end scatter tests', function() {
         .catch(fail)
         .then(done);
     });
+
+    it('should smoothly add/remove nodes tags with *ids* during animations', function(done) {
```

Comment:
and fixed in commit 9c35675

Before 9c35675
![peek 2017-05-23 14-09](https://cloud.githubusercontent.com/assets/6675409/26369033/7722baee-3fc1-11e7-9bad-838cf4174525.gif)


After:

![peek 2017-05-23 14-08](https://cloud.githubusercontent.com/assets/6675409/26368995/5e8b5298-3fc1-11e7-9133-25adba004d81.gif)